site/docs/components/button/examples.mdx

```diff
@@ -80,4 +87,17 @@ A button with the prop `disabled={true}` will suppress all functionality. If you
 Use full width buttons on mobile devices or smaller viewports, the button will take up the full width of its container.
 
 </LivePreview>
+  
+<LivePreview componentName="button" exampleName="Deprecated">
+
+## Deprecated
```

Comment:
This should probably be Variant (Deprecated) or something, labelling it deprecated makes it harder to tell at a glance what the example is